packaging/windows/get-convert-licenses.sh

```diff
@@ -27,3 +27,19 @@ function check_and_get_file() {
 check_and_get_file "gpl-3.0.rtf" "https://www.gnu.org/licenses/gpl-3.0.txt"
 check_and_get_file "ncul1.rtf" "https://app.netdata.cloud/LICENSE.txt"
 
+function create_unified_license() {
```

Comment:
Strictly speaking this is likely to not be ‘correct’ from a legal perspective. We need to be explicitly calling out what parts are under what license.

Reorganizing it so the NCUL comes first, and then putting something about ‘All other parts of the software are licensed under the GPL version 3.0’. Even that may not quite be correct though, we’re bundling and shipping a large chunk of MSYS2 on Windows and I’m not certain that 100% of the stuff we are bundling there has GPL compatible licenses (in which case those components that don’t need to be called out separately).